packages/@aws-cdk/aws-redshift/lib/cluster.ts

```diff
@@ -304,6 +305,13 @@ export interface ClusterProps {
    */
   readonly roles?: iam.IRole[];
 
+  /**
+   * A single AWS Identity and Access Management (IAM) role to be used as the default role for the cluster.
+   *
+   * @default - No default role is attached to the cluster.
+   */
+  readonly defaultRole?: iam.IRole;
```

Comment:
I'm absolutely fine with keeping this as a singleton, but given the roles prop is a list could this not also be a list? Is there a reason why a default singleton role would be preferable over a potential list of default roles?

Reply:
Redshift only allows the one default role per cluster. 
Assume your cluster has two IAM roles (role1 and role2). If role1 is default and then you modify the cluster to make role2 the default, role1 will still be a role for the cluster but no longer the default role.